### PR TITLE
Apply constraint layout

### PR DESCRIPTION
**Pull request by chibatching**
## Issue
- Apply constraint layout to view_session_cell.xml #139

## Overview (Required)
- Introduce constraint layout
- Apply constraint layout to view_session_cell.xml
- Update licenses.yml
- Re-generate licenses.html

## Links
- https://developer.android.com/reference/android/support/constraint/ConstraintLayout.html

## Screenshot
Before | After
:--: | :--:
 | 
 |

### CODE REVIEW

**Comment by konifar** [comment]
Cool!! 🆒

**Comment by chibatching** [comment]
I found little change, please see second screenshot diff.

Before, between category border and parent have little space. But I can't confirm why these are occurred.
Parent relative layout had 6dp or 4dp padding and category border had -6dp and -4dp margin.

If these space are intended, I'll add extra space between category boarder and parent layout.

**Comment by konifar** [comment]
CI failed 🤔 

```
> Failed to install the following SDK components:
  [Solver for ConstraintLayout 1.0.0-beta4, ConstraintLayout for Android 1.0.0-beta4]
  Please install the missing components using the SDK manager in Android Studio.
```

File: app/src/main/assets/licenses.html [hunk 1]
```diff
@@ -540,6 +540,252 @@ <h2 class="inline">9. Accepting Warranty or Additional Liability.</h2>
     </div>
     <div class="library">
       <!-- https://opensource.org/licenses/Apache-2.0 -->
+      <h1 class="title">DeployGate SDK</h1>
+      <p class="notice">Copyright &copy; DeployGate, tnj, henteko. All rights reserved.</p>
```
Comment (chibatching) [inline app/src/main/assets/licenses.html]: In older PR, deploygate was added to license.yml but generateLicensePage task was not executed.

**Comment by chibatching** [comment]
Oops, constraint layout cannot be installed using normal SDK manager (only in Android Studio).
We should download directly from google repository. I'll edit circle.yml, please wait.

### Link
- https://github.com/chibatching/docker-android-sdk/blob/master/Dockerfile#L31-L34

File: app/src/main/res/layout/view_session_cell.xml [hunk 2]
```diff
@@ -32,22 +26,24 @@
 
         <View
                 android:id="@+id/categoryBorder"
-                android:layout_width="match_parent"
+                android:layout_width="0dp"
                 android:layout_height="3dp"
-                android:layout_marginEnd="-6dp"
-                android:layout_marginLeft="-6dp"
-                android:layout_marginRight="-6dp"
-                android:layout_marginStart="-6dp"
```
Comment (chibatching) [inline app/src/main/res/layout/view_session_cell.xml]: ConstraintLayout doesn't support negative margin. We should add positive margin to other views.

**Comment by chibatching** [comment]
umm

> Your build has exceeded the memory limit of 4G on 1 container. The results of this build are likely invalid. We have taken a snapshot of the memory usage at the time, which you can find in a build artifact named `memory-usage.txt`. The RSS column in this file shows the amount of memory used by each process, measured in kilobytes.

**Comment by chibatching** [comment]
Fix CI failure!

**Comment by konifar** [comment]
Thanks! I'm so excited to use ConstraintLayout! 💪

**Pull request merged**